tensorflow/python/module/module.py

```diff
@@ -355,8 +352,7 @@ def _flatten_module(module,
       if predicate(leaf):
         if with_path:
           yield module_path + leaf_path, leaf
-        else:
```

Comment:
I'm not sure I agree that removing this else improves readability. I'd recommend adding it back.